content/kubernetes-backing-services/monitoring/kube-prometheus.md

```diff
@@ -0,0 +1,134 @@
+metrics, make alerts based on it and display the metric in Grafana charts.
+
+## Collection custom metrics
+
+1. Check the [list of available exporters](https://prometheus.io/docs/instrumenting/exporters) to make sure there isn't already an Exporter that will meet your needs. For example, there are exporters that will work with [JMX](https://github.com/prometheus/jmx_exporter) out-of-the-box.
+2. If there is no exporter, then you'll need to [write your own](https://prometheus.io/docs/instrumenting/writing_exporters/) exporter that will provide required [metrics in correct format](https://prometheus.io/docs/instrumenting/exposition_formats/)
+3. Create a Kubernetes service that provides an [exporter http endpoint](https://kubernetes.io/docs/concepts/services-networking/service). We recommend then deploying this service using a Helm Chart.
+4. Register the exporter for service discovery by creating a Kubernetes resource of type `ServiceMonitor` that points to the service created in step 3.
+
+### Examples
+
+* Running Exporters with [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/user-guides/running-exporters.html)
+* Custom configuration of [Service Monitor](https://coreos.com/operators/prometheus/docs/latest/custom-configuration.html)
+* Helm chart that exposes [Etcd metrics](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-kube-etcd)
+
+## Custom dashboards with Grafana
+
+Before start read basic introduction about grafana [dashboards and panels](http://docs.grafana.org/features/panels/graph/)
+
+* Sign-in to the Kubernetes Portal. Then navigate to "Grafana".
+{{< img src="/assets/kube-prometheus-4f06031c.png" title="https://grafana.portal.us-west-2.staging.example.com" >}}
+* Login to Grafana (default credentials are `admin/admin`)
+{{< img src="/assets/kube-prometheus-f5f4472c.png" title="https://grafana.portal.us-west-2.staging.example.com/login?redirect=%2F" >}}
+{{< img src="/assets/kube-prometheus-d0d17131.png" title="Grafana homepage authentificated user" >}}
+* Create dashboard
+{{< img src="/assets/kube-prometheus-ce3e9690.png" title="https://grafana.portal.us-west-2.staging.aws.popchest.io/dashboard/new?orgId=1" >}}
+Follow [Grafana dashboards documentations](http://docs.grafana.org/reference/templating/), read nice post [How to Create Your Own Grafana Dashboards](https://mapr.com/support/s/article/How-to-Create-Your-Own-Grafana-Dashboards?language=en_US) or watch
+[Beginners guide to building dashboards](https://www.youtube.com/watch?v=&index=7&list=PLDGkOdUX1Ujo3wHw9-z5Vo12YLqXRjzg2)
+{{< youtube sKNZMtoSHN4 >}}
+
+* [Export dashboard to JSON](http://docs.grafana.org/reference/export_import/#exporting-a-dashboard)
+
+Next steps depends of way you installed `kube-prometheus`
+
+### Installed with Master Helmfile
+
+* Put the exported content to the `/conf/kops/custom_values/kube-prometheus.grafana.dashboards.yaml`
```

Comment:
I think I'd prefer using `/conf/kops/values/`as I don't see the need to overly namespace values.